packages/components/src/Popover/Popover.css

```diff
@@ -20,20 +30,8 @@
   text-align: right;
 }
 
-.popover {
-  z-index: var(--elevation-tooltip);
-  width: max-content;
-  max-width: var(--popover--width);
-  box-shadow: var(--shadow-base);
-  border: var(--border-base) solid var(--color-border);
-  border-radius: var(--radius-base);
-  font-size: var(--typography--fontSize-base);
-  line-height: normal;
-  background: var(--color-surface);
-}
-
 .popover > * {
-  --public-content--padding: var(--card--base-padding);
+  --public-content--padding: var(--popover-content--base-padding);
```

Comment:
this is a strange css rule. I think I get what it's doing but there's gotta be a more elegant way to do this.

don't need to do it in this PR. just wanted to mention is because it feels like something that can be improved.